app/models/ansible_role.rb

```diff
@@ -6,7 +6,8 @@ class AnsibleRole < ApplicationRecord
   include Authorizable
 
   self.include_root_in_json = false
-  validates :name, :presence => true, :uniqueness => true
+  validates :name, :presence => true, :uniqueness => { scope: :organization_id }
```

Comment:
Having taxonomies behave completely differently for Ansible roles than all other models could cause a lot of confusion. Are we sure we want to do this?

Will there be a follow-up for Locations?

Reply:
This was part of a discussion, that taxonomies should behave like this for everything and us trying this concept (Katello concept) in here and if it would be easier to have single org and strict split of resources by orgs

Reply:
Ok, let's go ahead and see.